docs/installation.rst

```diff
@@ -4,72 +4,53 @@
 Installation
 ============
 
-There are two installation types, a production installation, which is static, and a development installation, which is editable.
+For Users
+=========
 
+To install the latest release from PyPI:
 
-Preparation
------------
+.. code-block:: bash
 
-To install meteodata-lab you need a miniconda installation. You can either set up your miniconda installation manually or use the script `tools/setup_miniconda.sh`, which will download and install the latest version of miniconda.
+   pip install meteodata-lab
 
+Optional Extras
+---------------
 
-Installation of dependencies
-----------------------------
+To install optional extras:
 
-Dependencies are handled by the conda package manager. The goal of this step is to set up a conda environment according to the requirements of meteodata-lab. Note that by design, there are some dependencies already when you start developing the package, as the environment includes linters and other development tools.
+.. code-block:: bash
 
-The dependencies are handled in requirement files. Free installations are based on the `requirements/requirements.yaml` file, where the top-level dependencies of the package are listed. Pinned installations are based on exported environments and stored in the file `requirements/environment.yaml`.
+   pip install "meteodata-lab[polytope,regrid]"
 
-Environments (based on either unpinned or pinned requirements) are handled by the script `tools/setup_env.sh`. The optional flag `-u` stands for unpinned installation:
+**Note**: The ``fdb`` extra is currently disabled because its dependency ``pyfdb`` is not available on PyPI. As an alternative, the development setup can be used.
```

Comment:
```suggestion
.. note::
   The ``fdb`` extra is currently disabled because its dependency ``pyfdb`` is not available on PyPI. As an alternative, the development setup can be used.
```